Ira/movies-app/src/components/moviePageHeader.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {Component} from 'react';
+import {Link} from "react-router-dom";
+import moviesData from './moviesData';
+import MovieCard from './movieCard';
+
+
+
+const MoviePageHeader = () => {
```

Comment:
I don't really like this header. I would suggest to split it into separate components. 
**Header** that renders the logo and search link by condition
**MovieDescription** that goes below the Header and renders movie info. 
OR 
You can leave like it is, BUT still you have to move code to MovieDescriptions and use it here.